README.md

```diff
@@ -45,12 +45,14 @@ If you have a question on how to use plugin (and the [docs](https://docs.sonarqu
 To request a new feature, please create a new thread in [SonarSource Community Forum](https://community.sonarsource.com/). Even if you plan to implement it yourself and submit it back to the community, please start a new thread first to be sure that we can follow up on it.
 
 ### Pull Request (PR)
-To submit a contribution, create a pull request for this repository. Please make sure that you follow our [code style](https://github.com/SonarSource/sonar-developer-toolset) and all [tests](#testing) are passing (Travis build is created for each PR).
+To submit a contribution, create a pull request for this repository. Please make sure that you follow our [code style](https://github.com/SonarSource/sonar-developer-toolset) and all [tests](#testing) are passing.
 
 ### Custom Rules
 If you have an idea for a rule but you are not sure that everyone needs it you can implement a [custom rule](https://docs.sonarqube.org/latest/analysis/languages/php/) available only for you.
 
 #### Custom Rules API Changes
+- **3.32**, support of PHP 8.3:
+    * Custom rule examples have been moved in this project, in folder [php-custom-rules](php-custom-rules)
```

Comment:
The change you mentioned is not related to the custom rules API. Please check the ticket again.